diff --git a/src/rcamera.h b/src/rcamera.h
index e6f5bba2..43a0c5ed 100644
--- a/src/rcamera.h
+++ b/src/rcamera.h
@@ -199,7 +199,7 @@ RLAPI Matrix GetCameraProjectionMatrix(Camera *camera, float aspect);
 //----------------------------------------------------------------------------------
-#define CAMERA_MOVE_SPEED                           5.4f       // Units per second
+#define CAMERA_MOVE_SPEED                           54.0f       // Units per second
 #define CAMERA_ROTATION_SPEED                       0.03f
-#define CAMERA_PAN_SPEED                            2.0f
+#define CAMERA_PAN_SPEED                            20.0f
 
 // Camera mouse movement sensitivity
 #define CAMERA_MOUSE_MOVE_SENSITIVITY               0.003f
